Main/ClientApp/src/Components/SpotifyCallbackPage.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { Redirect } from "react-router-dom";
+import { UrlHash } from "../Infrastructure/UrlUtilities";
+import Cookies from "js-cookie";
+import { useCookie } from "../Infrastructure/CustomReactHooks";
+
+export default function SpotifyCallbackPage() {
+  const hash = window.location.hash;
+  let params = UrlHash(hash);
+
+  Cookies.set("spotifyUserToken", JSON.stringify(params.get("access_token")));
```

Comment:
What happens when a spotify-authenticated logs out of the app and another one logs in? I think the cookie along with the token will stay and the new user will be logged in on the other's person account.
If that's a case, consider adding userId to the cookie or deleting the cookie on logout (though I think the first option is better)